src/cocotb/regression.py

```diff
@@ -780,7 +780,7 @@ def _record_test_failed(
             rest = ""
         else:
             rest = f": {msg}"
-        self.log.info(
+        self.log.warning(
             "%s %sfailed%s%s",
             self._test.fullname,
```

Comment:
This is a little awkward looking. https://github.com/cocotb/cocotb/actions/runs/13363291841/job/37351967608#step:24:3189

```suggestion
            "%s%s %sfailed%s%s",
            stop_hilight,
            self._test.fullname,
```